string name
string aux_info
bool terminate_current_task
string[] model_names
